tests/Behat/README.md

```
@@ -64,34 +72,24 @@ A new Redmine version could introduce new REST-API endpoints.
-
-```
-    @since60000
-    Scenario: Using a new feature
-        Given I have a "NativeCurlClient" client
-        And I create a project with name "Test Project" and identifier "test-project"
-        [...]
-```
-
 2. Exclude the tag from the specific suite in the `behat.yml` (note the `~` prefix):
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)